metricbeat/module/logstash/test_logstash.py

```diff
@@ -25,6 +25,7 @@ def test_node(self):
         self.check_metricset("logstash", "node", self.get_hosts(), self.FIELDS + ["process"])
 
     @unittest.skipUnless(metricbeat.INTEGRATION_TESTS, "integration test")
+    @unittest.skip("flaky test: https://github.com/elastic/beats/issues/26432")
```

Comment:
It had been already skipped some lines below with `unittest.skip()`.... but good to have the link to the issue!